### PR TITLE
Add quickstart for 002

### PR DESCRIPTION
**Pull request by MarkDaoust**

### CODE REVIEW

File: quickstarts/New_in_002.ipynb [hunk 1]
```diff
@@ -0,0 +1,1351 @@
+{
```
Comment (pcoet) [inline quickstarts/New_in_002.ipynb:1]: <p>"and and" -&gt; "and"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (pcoet) [inline quickstarts/New_in_002.ipynb:1]: <p>"in teh arguments" -&gt; "in the arguments"</p><p>"python" -&gt; "Python"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Review by pcoet — APPROVED** [review]

File: quickstarts/New_in_002.ipynb (hunk 1 above)
Comment (MarkDaoust) [reply quickstarts/New_in_002.ipynb:1]: Fixed.

Comment (MarkDaoust) [reply quickstarts/New_in_002.ipynb:1]: Fixed.

**Pull request merged**